test/run

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+. common
+
+check_root
+
+test_setup
+
+# part of this code is copied from:
+# https://github.com/mlichvar/linuxptp-testsuite/blob/master/run
+passed=() failed=()
+
+[ $# -gt 0 ] && tests=($@) || tests=(rt-*[^_])
+
+for test in "${tests[@]}"; do
+	echo "$test ($[${#passed[@]} + ${#failed[@]} + 1]/${#tests[@]})"
+	./$test && passed=(${passed[@]} $test) || failed=(${failed[@]} $test)
+	echo
+done
+
+echo "SUMMARY:"
+echo "  TOTAL  $[${#passed[@]} + ${#failed[@]}]"
+echo "  PASSED ${#passed[@]}"
+echo "  FAILED ${#failed[@]}  (${failed[@]})"
```

Comment:
This kind of accounting makes more sense if one single application is being tested and if we are interested in knowing the total or ratio of passing/failing tests.

I'd change to summarize:
- PASSED: list of applications
- FAILED: list of applications
- SKIPPED: list of applications